au.org.access-nri/model/spack/environment/location/README.md

```diff
@@ -0,0 +1,14 @@
+# ACCESS-OM2 Checksums
```

Comment:
```suggestion
# Spack build and deployment metadata
```

Or something ... not sold on that suggestion TBH

Reply:
Resolved